### ISSUE TITLE
terraform credhub support for secrets creation is ko

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#513]
### Expected behavior
As a platform editor, i want to be able to create credhub entries with COA terraform support

### Observed behavior
The credhub creation fails with v42, COA 4

```
 error(s) occurred:
* credhub_generic.openstack_networks_net-kubo: 1 error(s) occurred:
* credhub_generic.openstack_networks_net-kubo: The query parameter name is required for this request.: 

```

### references:
- v42 bumps credhub version
- might require https://github.com/orange-cloudfoundry/terraform-provider-credhub/releases/tag/v0.13.0

Note on v42 terraform plugins versions:

```
Terraform v0.11.7
+ provider.cloudfoundry v0.11.2
+ provider.credhub v0.12.1
+ provider.flexibleengine v1.2.0
+ provider.openstack v1.10.0
```

**Comment by poblin-orange** [comment]
induced by credhub bump

**Comment by o-orand** [comment]
fixed in COA 4.0.2 (see https://github.com/orange-cloudfoundry/cf-ops-automation/commit/20d3376a0441545a6a625c7234acb6e5a9babc84)
